### PR TITLE
Eth abu dhabi

### PR DESCRIPTION
**Pull request by patcito**
Add information to the ETH Abu Dhabi hackathon and conference next month, registration already open.

More info: https://www.ethabudhabi.org/

### CODE REVIEW

File: src/data/community-events.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "title": "ETH Abu Dhabi",
+    "to": "https://www.ethabudhabi.org/",
+    "sponsor": null,
+    "location": "Abu Dhabi, UAE",
+    "description": "The Ethereum dev conference and hackathon in Abu Dhabi on everything DeFi, privacy, EVM scaling, layers 2, Account Abstraction and more with a focus on decentralization and community projects. We also organize a Demo Pitch Day with VCs.",
+    "startDate": "2023-11-27",
+    "endDate": "2024-27-29"
```
Comment (minimalsm) [inline src/data/community-events.json:9]:
```suggestion
    "endDate": "2023-11-29"
```

**Comment by patcito** [comment]
All good? 🙂@F3n4nd0gm

**Comment by patcito** [comment]
anything I can do? this event is in 3 weeks :pray:

**Review by corwintines — APPROVED** [review]

**Pull request merged**